mkdocs.yml

```diff
@@ -25,3 +26,5 @@ plugins:
   - search
 extra_css:
   - css/extra.css
+#nav:
+#- Welcome: index.md
```

Comment:
@ContentBrewmaster I can change the landing page menu item but then it disconnects the rest of the menus so for now I have commented out the code till the menu can be worked on.